src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -436,159 +477,160 @@ export default {
 					},
 				});
 			});
-		}
-	},
-	created() {
-		// Read the page data from the cache
-		let baseData = {};
-		try {
-			baseData = this.apollo.readQuery({
-				query: lendByCategoryQuery,
-				variables: {
-					basketId: cookieStore.get('kvbskt'),
-				},
-			});
-		} catch (e) {
-			logReadQueryError(e, 'LendByCategory lendByCategoryQuery');
-		}
+		},
+		created() {
```

Comment:
@go-bears You'll need to add in one more closing bracket `},` above the "created" method. Then remove one of the closing brackets directly after the "beforeDestroy" method. 

As it stands now you've nested the "created" method within the "apollo" method whereas they should be siblings.

Reply:
thanks for helping me chase that down!